### PR TITLE
Suggest some improvements for UG

### PR DESCRIPTION
**Pull request by jordankanghm**
As per above.

Also did some pagenation, pls help to see if its okay!

edit: there's still some more UG PRs so the pagenation gotta change later haha

### CODE REVIEW

**Review by tiongMax — APPROVED** [review]
lgtm!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -319,7 +334,7 @@ Restrictions:
 
 Examples:
 * `eshow 7`
-  * result in a pop-up window appearing, listing all details of the event at index `7`.
+  * Creates a pop-up window containing all details of the event at index `7`.
```
Comment (ruo-x) [inline docs/UserGuide.md:337]: i think `shows` wld be a better word

**Review by ruo-x — COMMENTED** [review]
nice just one very minor issue, also the parameter names can j ignore hor coz ltr we will merge brendan PR

**Review by brein62 — APPROVED** [review]
insane

**Pull request merged**